core/src/main/c/freebsd/files.c

```diff
@@ -190,7 +168,33 @@ JNIEXPORT jlong JNICALL Java_io_questdb_std_Files_getLastModified
 #endif
 }
 
-#if defined(__APPLE__)
+#ifdef __APPLE__
+
+JNIEXPORT jboolean JNICALL Java_io_questdb_std_Files_allocate
```

Comment:
this is a copy from shared ?

Reply:
No, it's a copy from `linux/files.c`.

Reply:
Oh, you're asking about OS X implementation. Yes, I moved it from `shared/files.c`.